python-requirements.txt

```diff
@@ -29,6 +31,8 @@ pytest
 pytest-timeout
 pyyaml
 tabulate
+termcolor
+tqdm
```

Comment:
Could you use `enlighten` instead, which does the same thing as `tqdm`? Would be good to be able to minimize external dependencies.

Reply:
Fixed.